src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -1,19 +1,28 @@
 import React, { FC, useState, FormEvent, ChangeEvent } from "react";
-import { useAppDispatch } from "../../hooks/hooks";
-import { addTodo } from "../../store/reducers/UserSlice";
+import { useAppDispatch, useAppSelector } from "../../hooks/hooks";
+import { addTodo, checker } from "../../store/reducers/UserSlice";
 import ModalWindow from "../ModalWindow/ModalWindow";
 import InputGroup from "react-bootstrap/InputGroup";
 import s from "./TodoHeader.module.scss";
 
 const TodoHeader: FC = () => {
+  const filtValue = useAppSelector((state) => state.todos.filtValue);
   const dispatch = useAppDispatch();
   const [input, setInput] = useState("");
 
+  enum Сondition {
```

Comment:
The same - why do you duplicate code?

Reply:
I know , sorry, at first I thought that enum cannot be exported